cryptography/src/androidMain/kotlin/com/wire/kalium/cryptography/ProteusClientCryptoBoxImpl.kt

```diff
@@ -74,12 +78,12 @@ class ProteusClientCryptoBoxImpl constructor(
     //  parse it content we can consider changing it to a simple check if the session file exists on the local storage or not
     //  or rename it to doesValidSessionExist
     override suspend fun doesSessionExist(sessionId: CryptoSessionId): Boolean =
-        withContext(ioContext) {
+        withContext(defaultContext) {
             box.tryGetSession(sessionId.value)?.let { true } ?: false
         }
 
     override suspend fun createSession(preKeyCrypto: PreKeyCrypto, sessionId: CryptoSessionId) {
-        withContext(ioContext) {
+        withContext(defaultContext) {
```

Comment:
same here

Reply:
i did talk to @ohassine  and there is no need to use 2 different contexts here, we can run everything on and make sure it is Synchronize to avoid concurrent execution